HW_7/view.py

```diff
@@ -0,0 +1,145 @@
+# from logger import log
+# from random import randint
+
+# @log
+# def greetings():
+#     ''' Приветствие '''
+#     print('\nЯ Справочник\n')
+
+# @log
+# def menu() -> str: 
+#     '''Меню справочника '''
+#     return input('Выберите пункт меню: \n'
+#                  '1 - показать справочник\n'
+#                  '2 - добавить запись\n'
+#                  '3 - выход\n')
+
+# @log
+# def print_book(data: list):
+#     for el in data:
+#         print(f'Имя: {el["first_name"]} | Фамилия: {el["last_name"]}'
+#                 f' | Телефон: {el["phone_number"]} | Дата рождения: {el["birthday"]}'
+#                 f' | Место работы: {el["workplace"]} | Id: {el["id"]}')
+#     if not data:
+#         print('Нет данных для отображения')
+
+# @log
+# def input_data():
+#     book = {}
+#     book["first_name"] = input('Введите имя: ')
+#     book["last_name"] = input('Введите фамилия: ')
+#     book["phone_number"] = input('Введите телефон: ')
+#     book["birthday"] = input('Введите дату рождения: ')
+#     book["workplace"] = input('Введите место работы: ')
+#     book["id"] = randint(1, 150)
+
+from logger import log
+
+
+@log
+def greatings():
+    '''Вывод приветствия.'''
+
+    print('Справочник')
+    pass
+
+
+@log
+def menu() -> int:
+    """
+    Вывод меню, запрос данных от пользователя.
+    :return:
+    0 - Выход
+    1 - Загрузить из файла и вывести на экран
+    2 - Добавить новую запись
+    3 - Редактировать запись по id
+    4 - Поиск по фамилии
+
```

Comment:
Так как это все попадет в лог, запись не самая удачная, слишком много инфы
```suggestion
    вывод меню
```